### DISCUSSION TITLE
Load pack not disappearing in React

### DISCUSSION THREAD

**Discussion by ViniciusAAssuncao** [jeffreylanters/react-unity-webgl#330] [Question and Answer]
I'm using the native state of WebGL in React which concerns a loading bar before the game starts;   
 const [progression, setProgression] = useState(0);

    useEffect(function () {
        unityContext.on("progress", function (progression) {
            setProgression(progression);
        });
    }, []);

The implementation is simple, however I'm facing a problem just as simple, but that seems insoluble to me. When the progress reaches 1 and returns true allowing the game to run, the loading bar does not disappear, in this case the element contained in;
<p className={GameStyles.progressBar}>Loading {parseInt(progression * 100)}%</p> (I modified it a bit).

How can I resolve this?

**Comment by jeffreylanters** [comment]
In case you're still looking for an answer, or someone else is running into this problem. There is also a built-in event `loaded` which is triggered when the Unity application is starting. You can use this event in order to for example hide the loading overlay you've created.
